src/plugins/discover/public/application/view_components/canvas/top_nav.tsx

```diff
@@ -89,6 +89,10 @@ export const TopNav = ({ opts }: TopNavProps) => {
       useDefaultBehaviors
       setMenuMountPoint={opts.setHeaderActionMenu}
       indexPatterns={indexPattern ? [indexPattern] : indexPatterns}
+      // TODO after
```

Comment:
added an item here to follow up on this:

Reply:
thanks